libraries/botbuilder-middleware-aws-comprehend/src/engine.ts

```diff
@@ -0,0 +1,128 @@
+import { Engine } from '@botbuildercommunity/middleware-engine-core';
+import * as aws from 'aws-sdk';
```

Comment:
`import { Comprehend } from 'aws-sdk'` is not an option? I like that style more if possible.

Reply:
Changed.